packages/next-codemod/bin/upgrade.ts

```diff
@@ -134,6 +134,13 @@ export async function runUpgrade(
     await runTransform(codemod, process.cwd(), { force: true })
   }
 
+  // Release https://github.com/vercel/next.js/releases/tag/v14.3.0-canary.45
+  // PR https://github.com/vercel/next.js/pull/65058
+  if (compareVersions(targetNextVersion, '14.3.0-canary.45') >= 0) {
```

Comment:
This still applies to React 19 -> React 19 codemods. It needs the same logic as we use for the other codemods. Only if you cross the .45 barrier should we suggest this codemod. Not just if you upgrade to something above .45 e.g. .46 -> .47